### PR TITLE
validation CGU et +18

### PR DESCRIPTION
**Pull request by AnassAmh**

### CODE REVIEW

File: src/app/pages/register-page/register-page.component.ts [hunk 1]
```diff
@@ -13,10 +13,10 @@ import { Router } from '@angular/router';
 })
 export class RegisterPageComponent {
   registerForm: FormGroup;
-  user?:User;
-  router=inject(Router)
+  user?: User;
+  router = inject(Router);
 
-  authetificationService=inject(AuthenticationService)
+  authetificationService = inject(AuthenticationService);
```
Comment (OCR-VH-DevWeb) [inline src/app/pages/register-page/register-page.component.ts:19]: Pay attention to spelling "authentification"

File: src/app/pages/register-page/register-page.component.ts [hunk 2]
```diff
@@ -26,12 +26,13 @@ export class RegisterPageComponent {
       pseudo: ['', [Validators.required]],
       emailAdress: ['', [Validators.required, Validators.email]],
```
Comment (OCR-VH-DevWeb) [inline src/app/pages/register-page/register-page.component.ts:27]: Same "mailAddress"

File: src/app/pages/register-page/register-page.component.ts [hunk 3]
```diff
@@ -26,12 +26,13 @@ export class RegisterPageComponent {
...
       confirmationMail: ['', [Validators.required, Validators.email]],
-      password: ['', [Validators.required, Validators.minLength(8)]],
-      confirmationMotDePasse: ['', [Validators.required]]
+      password: ['', [Validators.required, Validators.minLength(8), this.passwordValidator]],
+      confirmationMotDePasse: ['', [Validators.required]],
```
Comment (OCR-VH-DevWeb) [inline src/app/pages/register-page/register-page.component.ts:30]: No french for the classes names please thanks => passwordConfirmation

File: src/app/pages/register-page/register-page.component.ts [hunk 4]
```diff
@@ -51,20 +52,20 @@ export class RegisterPageComponent {
...
-
     if (this.registerForm.valid) {
-      this.user=new User(
-        this.registerForm.value.pseudo ,
+      this.user = new User(
+        this.registerForm.value.pseudo,
         this.registerForm.value.lastName,
         this.registerForm.value.name,
         this.registerForm.value.password,
         this.registerForm.value.emailAdress,
```
Comment (OCR-VH-DevWeb) [inline src/app/pages/register-page/register-page.component.ts:61]: Same "mailAddress"

File: src/app/pages/register-page/register-page.component.ts [hunk 5]
```diff
@@ -51,20 +52,20 @@ export class RegisterPageComponent {
...
         this.registerForm.value.birthDate,
-      )
-      this.authetificationService.register(this.user).subscribe()
-      this.router.navigateByUrl('/')
-
+      );
+      this.authetificationService.register(this.user).subscribe(() => {
```
Comment (OCR-VH-DevWeb) [inline src/app/pages/register-page/register-page.component.ts:64]: Same "authentification" ^^

**Review by OCR-VH-DevWeb — APPROVED** [review]
Make the corrections and it's good

**Review by OCR-VH-DevWeb — APPROVED** [review]
Approved

File: src/app/pages/news-character/news-character.component.scss [hunk 6]
```diff
@@ -143,7 +143,8 @@ h1 {
   .selection-box {
     width: 90%; 
     height: 50%;
-    margin-top: -240px;
+    margin-top: -230px;
+    //margin-bottom: 240px;
```
Comment (Benja-san) [inline src/app/pages/news-character/news-character.component.scss:147]: remove dead code

**Review by Benja-san — APPROVED** [review]

**Review by OCR-VH-DevWeb — APPROVED** [review]
Approved!!! Dead code removed thanks Anass

**Pull request merged**